pkg/server/render.go

```diff
@@ -159,9 +177,19 @@ func (*RenderHandler) mountRenderResponse(flame flamebearer.FlamebearerProfile,
 		Query:                 gi.Query.String(),
 		MaxNodes:              maxNodes,
 	}
+
```

Comment:
Did this just to reformat in such a way it makes sense for `/render`. Example:  it's redundant to have an `appName` repeated for each annotation.